app/controllers/api/v1/invoice_items_find_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::InvoiceItemsFindController < ApplicationController
+
+  def index
+    @invoice_items = InvoiceItem.where(find_params)
+  end
+
+  def show
+    @invoice_item = InvoiceItem.find_by(find_params)
+  end
+
+  private
+
+  def find_params
```

Comment:
I'm so happy we found this strong params workaround from Sal. This is beautiful.